trunk/source/CACSL2BoogieTranslator/src/de/uni_freiburg/informatik/ultimate/cdt/translation/implementation/base/ACSLHandler.java

```diff
@@ -677,6 +685,47 @@ private String lookupId(final IDispatcher main,
 		}
 	}
 
+	@Override
+	public Result visit(final IDispatcher main, final QuantifierExpression node) {
+		mQuantifiedVariables.beginScope();
+		final ILocation loc = mLocationFactory.createACSLLocation(node);
+		final List<VarList> quantifiedVars = new ArrayList<>();
+		final List<Expression> typeConstraints = new ArrayList<>();
+		for (final var decl : node.getVariables()) {
+			// For each quantified variable in the ACSL expression, create a corresponding Boogie variable and store it
+			// in the mQuantifiedVariables to be used when handling IdentifierExpressions.
+			final String name = decl.getName();
+			final CType cType = AcslTypeUtils.translateAcslTypeToCType(decl.getType());
+			final DeclarationInformation declInfo = new DeclarationInformation(StorageClass.QUANTIFIED, null);
+			final BoogieType boogieType = mTypeHandler.getBoogieTypeForCType(cType);
+			// TODO: Handle quantified variables on heap

```

Comment:
What exactly does this mean, and what would have to be done?

Reply:
Actually I don't know 😄
This wold happen, if we quantify over pointers and I am not sure about the ACSL semantics in that case.
Also, I don't think we support pointer types in ACSL properly, so we would probably crash earlier.
So maybe I can just replace the TODO with a comment that we don't support pointers for now and thus yield a `LocalLValue`.

Reply:
I am also confused what "quantified variables on heap" means. Is this a quantified pointer? 
But I agree, that sounds rather difficult. Perhaps we should crash as soon as we encounter it, i.e., throw UnsupportedSyntax .